index.js

```diff
@@ -375,6 +375,32 @@ export default () => {
       });
     }
     wearing = wear;
+
+    // Pistol is droped?
+    if (!wearing) {
+
+      const quaternion = new THREE.Quaternion(-0.7071068, 0, 0, 0.7071068); // Facing ground
```

Comment:
Better to do `.setFromAxisAngle(new THREE.Vector3(1, 0, 0), -Math.PI*0.5)` than random numbers and a comment.

Reply:
Fixed as requested